### PR TITLE
Reports mailto links clickable

### PR DESCRIPTION
**Pull request by Lyrenhex**
**Submitted for PR following request from GCI mentor.**

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by shreyans800755** [comment]
@Lyrenhex 
Can you please squash both the commits ?

You can visit coala.io/rebase if you're not familiar with the process.

**Comment by Lyrenhex** [comment]
Tried to, not sure if it's been done right? Sorry if not.

**Comment by shreyans800755** [comment]
@Lyrenhex Looks like you're adding commits. And also, we don't want merge commits in history.

There should be single commit to achieve single objective. Here, it looks like you are also fixing some spacing issues. For that, you can have separate commit.

Make sure you have 2 commits in this PR (one for main task, other for spacing fix - optional). And no merge commits please.
When you are making incremental changes on existing commit you've done, you can use `git commit --amend` to modify top commit.

**Comment by Lyrenhex** [comment]
The spacing stuff appears to have been automatically done by my editor, so not sure what to do about those. Managed to squash everything now though :)

**Comment by mridubhatnagar** [comment]
Hey @Lyrenhex. 

Good work with the commit message. You will just have to add which issue does this pull request resolve. 

That is 

Closes link to the issue

**Comment by shreyans800755** [comment]
@Lyrenhex Great. Can you please update the commit message ? Doesn't look proper. One of the link is only pasted properly.

**Comment by Lyrenhex** [comment]
Apologies, fixed. Had accidentally half-pasted the same issue link twice.

**Comment by shreyans800755** [comment]
Still couple of things:
First line in the commit body is too small. 
And also replace `Fixes` -> `Closes` as this is not a bug.

**Comment by Lyrenhex** [comment]
Alright, sorry. Addressed. :)

**Comment by jayvdb** [comment]
this is https://github.com/coala/projects/issues/354

fyi, linking the PR to the issue cross-links them , so we can find the PR easier.

**Comment by blazeu** [comment]
You modify too many files that doesn't relate to the commit, only `report.md` is needed.

Also, the shortlog should be `report.md: ..`

File: _layouts/report.md [hunk 1]
```diff
@@ -22,7 +22,7 @@
                     <h4 class="left light">Google Summer of Code 2017 Work Product Submission </h4>
                 </div>
                 <br>
-            </div> 
+            </div>
```
Comment (blazeu) [inline _layouts/report.md]: Please do not modify lines that doesn't relate to the commit.

**Comment by jayvdb** [comment]
I checked on Netlify , using https://deploy-preview-430--coala-projects.netlify.com/gsoc/2017/statusreport/hemangkumar , and functionally the code is good. (surprise!)

Now just need to tidy it up

**Comment by jayvdb** [comment]
unack https://github.com/coala/projects/pull/430/commits/ba07e6aac222e176d9e96d9b0d4c92c58f91c942

File: _layouts/report.md [hunk 2]
```diff
@@ -127,7 +127,7 @@
                     </thead>
                     <tbody class="card activity">
                         {% for commits in page.activity %}
-                        {% for c in commits %} 
+                        {% for c in commits %}
```
Comment (yukiisbored) [inline _layouts/report.md]: Please don't do unnecessary changes, it's not the scope of this PR.
You can fix this later.

File: _layouts/report.md [hunk 3]
```diff
@@ -148,4 +148,4 @@
             <br><br>
         </div>
     </body>
-</html> 
+</html>
```
Comment (yukiisbored) [inline _layouts/report.md]: Please don't do unnecessary changes, it's not the scope of this PR.
You can fix this later.

File: reports/RJ722.md [hunk 4]
```diff
@@ -41,7 +41,7 @@ activity:
  - 1:
    - repo: vulture
    - link: https://github.com/jendrikseipp/vulture/commit/e5012c424fdba1a6aa0b5da8e9befdf2e98e0fc9
-   - details: > 
+   - details: >
```
Comment (yukiisbored) [inline reports/RJ722.md]: Please don't do unnecessary changes, it's not the scope of this PR.
You can fix this later.

File: reports/RJ722.md [hunk 5]
```diff
@@ -146,26 +146,26 @@ activity:
 
 <br>
 
-1.) In addition to the first line of unused code, Vulture now reports the last 
+1.) In addition to the first line of unused code, Vulture now reports the last
```
Comment (yukiisbored) [inline reports/RJ722.md]: Please don't do unnecessary changes, it's not the scope of this PR.
You can fix this later.

File: reports/RJ722.md [hunk 6]
```diff
@@ -146,26 +146,26 @@ activity:
...
 line of unused code. The results can now also be sorted according to their size.
 
 <br>
 
-2.) Other than the conventional dead code (defined but not used), vulture 
+2.) Other than the conventional dead code (defined but not used), vulture
```
Comment (yukiisbored) [inline reports/RJ722.md]: Please don't do unnecessary changes, it's not the scope of this PR.
You can fix this later.

File: reports/RJ722.md [hunk 7]
```diff
@@ -146,26 +146,26 @@ activity:
...
 now also finds unsatisfiable conditions in `if/while` statements. Also,
-any code appearing after `return`, `break`, `continue` and `raise` statements 
+any code appearing after `return`, `break`, `continue` and `raise` statements
```
Comment (yukiisbored) [inline reports/RJ722.md]: Please don't do unnecessary changes, it's not the scope of this PR.
You can fix this later.

File: reports/RJ722.md [hunk 8]
```diff
@@ -146,26 +146,26 @@ activity:
...
 is also reported as dead code.
 
 <br>
 
-3.) Confidence values are now reported with every result. Also, a 
-`min_confidence` parameter can be passed to Vulture. It then only reports 
+3.) Confidence values are now reported with every result. Also, a
+`min_confidence` parameter can be passed to Vulture. It then only reports
```
Comment (yukiisbored) [inline reports/RJ722.md]: Please don't do unnecessary changes, it's not the scope of this PR.
You can fix this later.

File: reports/RJ722.md [hunk 9]
```diff
@@ -146,26 +146,26 @@ activity:
...
 results for which `confidence` > `min_confidence`.
 
 <br>
 
-4.) The Vulture API has been extended and it now exhibits all characteristics 
-of dead code, like `confidence`, `first_lineno`, `last_lineno`, `message`, etc. 
+4.) The Vulture API has been extended and it now exhibits all characteristics
+of dead code, like `confidence`, `first_lineno`, `last_lineno`, `message`, etc.
```
Comment (yukiisbored) [inline reports/RJ722.md]: Please don't do unnecessary changes, it's not the scope of this PR.
You can fix this later.

File: reports/hemang.md [hunk 10]
```diff
@@ -34,9 +34,9 @@ email: hemangsk@gmail.com
 blog: http://hemangsk.github.io/stories.html
 activity:
  - 0:
-   - repo: coala-quickstart 
+   - repo: coala-quickstart
```
Comment (yukiisbored) [inline reports/hemang.md]: Please don't do unnecessary changes, it's not the scope of this PR.
You can fix this later.

File: reports/hemang.md [hunk 11]
```diff
@@ -34,9 +34,9 @@ email: hemangsk@gmail.com
...
    - link: https://github.com/coala/coala-quickstart/commits/master?author=hemangsk
-   - details: > 
+   - details: >
```
Comment (yukiisbored) [inline reports/hemang.md]: Please don't do unnecessary changes, it's not the scope of this PR.
You can fix this later.

File: reports/hemang.md [hunk 12]
```diff
@@ -46,7 +46,7 @@ activity:
  - 2:
    - repo: coala Projects
    - link: https://github.com/coala/projects/commits/master?author=hemangsk
-   - details: > 
+   - details: >
```
Comment (yukiisbored) [inline reports/hemang.md]: Please don't do unnecessary changes, it's not the scope of this PR.
You can fix this later.

File: reports/hemang.md [hunk 13]
```diff
@@ -179,7 +179,7 @@ activity:
  - 32:
    - repo: landing-frontend
    - link: https://github.com/coala/landing-frontend/commits/master?author=hemangsk
-   - details: > 
+   - details: >
```
Comment (yukiisbored) [inline reports/hemang.md]: Please don't do unnecessary changes, it's not the scope of this PR.
You can fix this later.

File: reports/hemang.md [hunk 14]
```diff
@@ -192,7 +192,7 @@ The coala Online module now has support for all the bears, bears with optional a
 
 <br>
 
-coala-quickstart has got a new running mode, allow-incomplete-sections which helps users to create a coafile without omitting bears with non optional settings. 
+coala-quickstart has got a new running mode, allow-incomplete-sections which helps users to create a coafile without omitting bears with non optional settings.
```
Comment (yukiisbored) [inline reports/hemang.md]: Please don't do unnecessary changes, it's not the scope of this PR.
You can fix this later.

File: reports/hemang.md [hunk 15]
```diff
@@ -216,11 +216,11 @@ It was a great learning experience overall, but a few challenges which proved to
 
 #### Work to be done
 
-The first task, is to get all the supporting tools like coala Online, coala-html and projects.coala.io need to incooperate user feedback on the changes user experience. 
+The first task, is to get all the supporting tools like coala Online, coala-html and projects.coala.io need to incooperate user feedback on the changes user experience.
```
Comment (yukiisbored) [inline reports/hemang.md]: Please don't do unnecessary changes, it's not the scope of this PR.
You can fix this later.

File: reports/hemang.md [hunk 16]
```diff
@@ -216,11 +216,11 @@ It was a great learning experience overall, but a few challenges which proved to
...
 
 <br>
 
-coafile_bot will be able to do wonders on GitLab as well once GitLab introduces Notification API. Initally I want to promote it as much as I can so that developers can have general awareness about linting, the tools available and how coala comes into the picture, and from there they can use coafile bot and then ultimately improve their git workflow. 
+coafile_bot will be able to do wonders on GitLab as well once GitLab introduces Notification API. Initally I want to promote it as much as I can so that developers can have general awareness about linting, the tools available and how coala comes into the picture, and from there they can use coafile bot and then ultimately improve their git workflow.
```
Comment (yukiisbored) [inline reports/hemang.md]: Please don't do unnecessary changes, it's not the scope of this PR.
You can fix this later.

**Comment by Lyrenhex** [comment]
Re-committed. Thanks :) I hope this time it's okay then?

**Comment by stellargo** [comment]
ack 713c4c9d00fdbcefe8322325e047bcf9018a894b

**Comment by Mixih** [comment]
@rultor merge

**Pull request merged**